changelog

```diff
@@ -1,6 +1,7 @@
 MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki
-V.Next
+Version 4.4.0
 ----------
+- [PATCH] Update common @11.0.0
```

Comment:
No common change?

Reply:
mmm..  not sure, what this means but common 11.0.0 is the latest common we published, and the common that use msal 4.4.0